proto/cosmwasm/wasm/v1/authz.proto

```diff
@@ -34,6 +47,18 @@ message ContractMigrationAuthorization {
       [ (gogoproto.nullable) = false, (amino.dont_omitempty) = true ];
 }
 
+// CodeGrant a granted permission for a single code
+message CodeGrant {
+  // CodeHash is the unique identifier created by wasmvm
+  // Wildcard "*" is used to specify any kind of grant.
+  bytes code_hash = 1;
```

Comment:
Is there a reason why this is not a string? All checksums are lower case hex string in the CosmWasm stack.

What kind of bytes are expected here? 32 raw bytes hash output or the hex string?

Reply:
yes, it was kept as bytes to keep consistency with other code_hash/checksum attributes in the proto files such as:
``` types.proto
message CodeInfo {
  // CodeHash is the unique identifier created by wasmvm
  bytes code_hash = 1;
  ```
  and
  ``` tx.proto
  message MsgStoreCodeResponse {
  // CodeID is the reference to the stored WASM code
  uint64 code_id = 1 [ (gogoproto.customname) = "CodeID" ];
  // Checksum is the sha256 hash of the stored code
  bytes checksum = 2;
}
```

Reply:
Thanks for pointing this out. I will be more careful in the future.
Maybe in future releases we can fix all these fields and have migrations for them. I don't think it will be too difficult.